tests/NCronJob.Tests/.editorconfig

```diff
@@ -1,3 +1,7 @@
 [*.cs]
 
-dotnet_diagnostic.IDISP017.severity = none # IDISP017: Prefer using
+# IDISP017: Prefer using
+dotnet_diagnostic.IDISP017.severity = none 
+
+# CA2211: Non-constant fields should not be visible
+dotnet_diagnostic.CA2211.severity = suggestion
```

Comment:
Sadly, the added TheoryData needs to be public

Reply:
In xunit v2 that wasn't the case, was it?

Reply:
I'm not sure it's related to the xunit upgrade.
That's a a built-in dotnet analyze warning.
However, it might be the first case of leveraging a TheoryData through a method in the codebase. Hence the warning and the silencing scoped to the Tests project